Updated few functions bool tag and return values to bool (#823)
openmultiplayer/web: docs/scripting/functions/ApplyActorAnimation.md

Apply an animation to an actor.

| Name       | Description                                                                                                                                                                                            |
| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| actorid    | The ID of the actor to apply the animation to.                                                                                                                                                         |

> | animlib[]  | The animation library from which to apply an animation.                                                                                                                                                |

Reviewer: ```suggestion
| const animationLibrary[]  | The animation library from which to apply an animation.                                                                                                                                                |
```

> | animname[] | The name of the animation to apply, within the specified library.                                                                                                                                      |

Reviewer: ```suggestion
| const animationName[] | The name of the animation to apply, within the specified library.                                                                                                                                      |
```

> | fDelta     | The speed to play the animation (use 4.1).                                                                                                                                                             |

Reviewer: ```suggestion
| Float:delta     | The speed to play the animation (use 4.1).                                                                                                                                                             |
```

> | loop       | If set to true, the animation will loop. If set to false, the animation will play once.                                                                                                                |

Reviewer: ```suggestion
| bool:loop       | If set to true, the animation will loop. If set to false, the animation will play once.                                                                                                                |
```

> | lockx      | If set to false, the actor is returned to their old X coordinate once the animation is complete (for animations that move the actor such as walking). true will not return them to their old position. |

Reviewer: ```suggestion
| bool:lockX      | If set to false, the actor is returned to their old X coordinate once the animation is complete (for animations that move the actor such as walking). true will not return them to their old position. |
```

> | locky      | Same as above but for the Y axis. Should be kept the same as the previous parameter.                                                                                                                   |

Reviewer: ```suggestion
| bool:lockY      | Same as above but for the Y axis. Should be kept the same as the previous parameter.                                                                                                                   |
```

> | freeze     | Setting this to true will freeze an actor at the end of the animation. false will not.                                                                                                                 |

Reviewer: ```suggestion
| bool:freeze     | Setting this to true will freeze an actor at the end of the animation. false will not.                                                                                                                 |
```
